docs/content/getting_started/contributing/documentation.rst

```diff
@@ -70,18 +70,44 @@ For example:
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you would like to see the documentation changes.
-In the repo's root directory, enter the following in your terminal.
+You can build the documentation pages locally to check how new changes would
+look like. First, make sure that you have :ref:`created and activated an
```

Comment:
```suggestion
You can build the documentation pages locally to see how the new changes will look. 
First, make sure that you have :ref:`created and activated an
```

Reply:
The original statement just sounded odd to me...

Also your `:ref:` link is broken